### PR TITLE
Explain *-hook and -q in renewal documentation

### PR DESCRIPTION
**Pull request by bmw**
Adds information about helpful flags to our renewal documentation.

### CODE REVIEW

**Comment by SwartzCr** [comment]
lgtm will merge once tests finish

**Comment by pde** [comment]
All invocations of renewal commands (including the `certonly` examples) should probably have `-n` attached to them.

File: docs/using.rst [hunk 1]
```diff
@@ -215,12 +215,25 @@ expire in less than 30 days. The same plugin and options that were used
...
+information about renewal hooks can be found by running
+``letsencrypt --help renew``.
+
 If you're sure that this command executes successfully without human
 intervention, you can add the command to ``crontab`` (since certificates
 are only renewed when they're determined to be near expiry, the command
-can run on a regular basis, like every week or every day); note that
-the current version provides detailed output describing either renewal
-success or failure.
+can run on a regular basis, like every week or every day). You may also
```
Comment (pde) [inline docs/using.rst]: "You may also want" -> "In that case you are likely to want"

**Comment by pde** [comment]
LGTM modulo minor revisions.

**Comment by bmw** [comment]
Ready for another review.

**Comment by pde** [comment]
LGTM; merge when green.

**Pull request merged**